CSharp/Tests/Microsoft.Bot.Connector.Teams.Tests.Shared/ConnectorExtensionTests.cs

```diff
@@ -106,6 +106,66 @@ public void ConnectorExtensions_Create1on1()
             Assert.IsTrue(conClient.Conversations.CreateOrGetDirectConversation(botAccount, userAccount, "TestTenantId").Id == "TestId");
         }
 
+        /// <summary>
+        /// Get paged teams conversation members test
+        /// </summary>
+        /// <returns>Task tracking operation.</returns>
+        [TestMethod]
+        public async Task ConnectorExtensions_GetTeamsPagedConversationMembersAsync()
+        {
+            TestDelegatingHandler testDelegatingHandler = new TestDelegatingHandler((request) =>
+            {
+                Assert.IsFalse(request.Headers.Contains("X-MsTeamsTenantId"));
+
+                StringContent stringContent = new StringContent(File.ReadAllText(@"Jsons\SampleResponseGetTeamsPaginatedConversationMembers.json"));
+                var response = new HttpResponseMessage(HttpStatusCode.OK)
+                {
+                    Content = stringContent
+                };
+                return Task.FromResult(response);
+            });
+
+            ConnectorClient conClient = new ConnectorClient(new Uri("https://testservice.com"), "Test", "Test", testDelegatingHandler);
+
+            var memberPagedResult = await conClient.Conversations.GetTeamsPagedConversationMembersAsync("TestConversationId", 2);
+            var members = memberPagedResult.Members;
+
+            Assert.IsTrue(members.Count() == 2);
+            Assert.IsFalse(string.IsNullOrEmpty(memberPagedResult.ContinuationToken));
+            Assert.IsFalse(members.Any(member => string.IsNullOrEmpty(member.ObjectId)));
+            Assert.IsFalse(members.Any(member => string.IsNullOrEmpty(member.Id)));
+            Assert.IsFalse(members.Any(member => string.IsNullOrEmpty(member.Name)));
+        }
+
+        /// <summary>
+        /// Get paged teams conversation members test
```

Comment:
nit: also we might need to update the summary here as it seems it is testing a single user?